Firestore/Example/Tests/SpecTests/FSTMockDatastore.mm

```diff
@@ -42,12 +47,12 @@ - (instancetype)initWithDatastore:(FSTMockDatastore *)datastore
                       credentials:(id<FSTCredentialsProvider>)credentials
                        serializer:(FSTSerializerBeta *)serializer NS_DESIGNATED_INITIALIZER;
 
-- (instancetype)initWithDatabase:(FSTDatabaseInfo *)database
+- (instancetype)initWithDatabase:(DatabaseInfo)database
```

Comment:
Could these be `const DatabaseInfo&`? Normally we pass `std::string` this way and this is more than one of those. If you agree, here and throughout.